pkg/operator/v410_00_assets/bindata.go

```diff
@@ -138,6 +139,45 @@ func v410AlertsApiUsageYaml() (*asset, error) {
 	return a, nil
 }
 
+var _v410AlertsAuditErrorsYaml = []byte(`apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: audit-errors
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+  - name: apiserver-audit
+    rules:
+    - alert: AuditLogError
+      annotations:
+        summary: |-
+          Incident response teams rely on the audit logs to detect malicious activity.
+          If an API Server is unable to succesfully write an event to an audit log,
+          this might hinder the incident response team in dealing with a security
+          incident.
+        description: An API Server had an error writing to an audit log.
+      expr: |
+        sum by (apiserver,instance)(rate(apiserver_audit_error_total{apiserver=~".+-apiserver"}[5m])) / sum by (apiserver,instance) (rate(apiserver_audit_event_total{apiserver=~".+-apiserver"}[5m])) > bool 0
+      for: 1m
+      labels:
+        severity: critical
```

Comment:
this will wake up SREs in the night. I don't think this is the right thing to do. Generally: A `critical` alert will wake up people in the night, while a `warning level signals an upcoming issue and most likely implies a ticket to be generated automatically.

I am voting for `warning` level here.

/cc @RiRa12621

Reply:
@s-urbaniak this one is tricky because different deployments will have different requirements. A FedRAMP moderate compliant setup will need immediate intervention (whatever that means depends on the organization), while a DOD Impact level 5 will require an immediate shutdown of the system when audit logs fail... a less critical system will most likely not have such requirements though...

This is not a particularly common occurrence, hence why I went for critical already. I'm happy to get more input from folks though.

Reply:
changed it to a `warning`